zebra-rpc/src/methods/get_block_template_rpcs/get_block_template.rs

```diff
@@ -139,6 +140,37 @@ where
     Ok(chain_info)
 }
 
+/// Returns the transactions that are currently in `mempool`.
+///
+/// You should call `check_synced_to_tip()` before calling this function.
+/// If the mempool is inactive because Zebra is not synced to the tip, returns no transactions.
+//
+// TODO: return an error if the mempool is inactive?
```

Comment:
I think `check_synced_to_tip()` would return an error before this is called if the mempool is inactive.

Reply:
Removed in 854a4f1760.

Reply:
Removed in 854a4f1760.